src/Carousel.vue

```diff
@@ -476,9 +496,54 @@ export default {
     getPreviousPage() {
       if (this.currentPage > 0) {
         return this.currentPage - 1;
+      } else if (this.infinite) {
+        return this.getPreviousInfinitePage();
       }
       return this.loop ? this.pageCount - 1 : this.currentPage;
     },
+    /**
+     * Moves the HTMLElements around to allow infinite scrolling going forward.
+     * @return {Number} The index of the next page
+     * */
+    getNextInfinitePage() {
+      const carouselInner = this.$refs["VueCarousel-inner"],
+        firstChild = carouselInner.children[0],
+        lastChild = carouselInner.children[carouselInner.children.length - 1];
+
+      lastChild.after(firstChild.cloneNode(true));
+      this.infiniteMove = { direction: "next", newItem: firstChild };
+      return this.currentPage + 1;
+    },
+    /**
+     * Moves the HTMLElements around to allow infinite scrolling going back.
+     * @return {Number} The index of the next page
+     * */
+    getPreviousInfinitePage() {
+      const carouselInner = this.$refs["VueCarousel-inner"],
+        firstChild = carouselInner.children[0],
+        lastChild = carouselInner.children[carouselInner.children.length - 1];
+
+      this.hideTransition = true;
+      this.justifyContent = "flex-start";
+      this.forcedOffset = -250;
```

Comment:
These hardcoded offsets will cause issues for anyone not setting the width to 500px. I tried the feature in vue play and set the width to something other than 500 and I was getting a weird jumping behaviour. I think these should be more dynamic.